Lib/asyncio/proactor_events.py

```diff
@@ -170,11 +184,25 @@ def resume_reading(self):
         if self._loop.get_debug():
             logger.debug("%r resumes reading", self)
 
-    def _loop_reading(self, fut=None):
+    def _loop_reading__on_eof(self):
```

Comment:
Is two underscore in name intentional?
We never use this code style in asyncio

Reply:
It's just a bit more readable this way, since it's one "family" of functions (all three of them are combined in one "mega" function).  I can replace with one underscore, but I think it's OK this way too. Up to you.

Reply:
Ok, agree